macheteLogger/src/main/java/com/virtuslab/logger/MacheteLogger.java

```diff
@@ -0,0 +1,34 @@
+package com.virtuslab.logger;
+
+import java.util.function.Supplier;
+
+import kr.pe.kwonnam.slf4jlambda.LambdaLogger;
+
+public class MacheteLogger implements IMacheteLogger {
```

Comment:
Technically, this might be/will be used from non-machete-aware modules (`binding`, `branchLayoutImpl`, `gitCoreJGit`), so I'd give it a machete-agnostic name... maybe `ClassAndMethodPrefixingLambdaLogger`? A bit cumbersome though :/

Reply:
And a module... `logging`?

Reply:
Changed to `PrefixedLambdaLogger` for class and `logging` for module